tests/unit/server_tests/test_pwd_file.py

```diff
@@ -93,7 +87,7 @@ def test_given_username_but_no_password_expect_parsing_fails(self):
                        "{} {}".format(login, pwd))
 
         with self.assertRaises(RuntimeError) as cm:
-            app = TabPyApp(self.config_file.name)
+            _ = TabPyApp(self.config_file.name)

```

Comment:
no need for the underscore, just call TabPyApp without assigning it to anything

Reply:
Fixed